src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -373,6 +373,30 @@ public sealed override bool HasUnsupportedMetadata
 
         #endregion
 
+        /// <summary>
+        /// Is this a field of a tuple type?
+        /// </summary>
+        public virtual bool IsTupleField
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        /// <summary>
+        /// If this is a field of a tuple type, return corresponding underlying field from the
+        /// tuple underlying type. Otherwise, null. In case of a mulformed underlying type
```

Comment:
Thanks, I'll address this in the following change.

---
In reply to: [61784205](https://github.com/dotnet/roslyn/pull/11006#discussion_r61784205) [](ancestors = 61784205)